.github/workflows/e2e-chrome.yml

```diff
@@ -72,6 +72,8 @@ jobs:
       matrix-total: ${{ strategy.job-total }}
 
   test-e2e-chrome-vault-decryption:
+    # This if statement is equivalent to IS_FORK
+    if: ${{ !(github.event.pull_request.head.repo.fork || github.event.repository.fork) }}
```

Comment:
:information_source: sounds good! I think it makes sense to disable it just in forks because of this:

https://github.com/MetaMask/metamask-extension/issues/35265#issuecomment-3210653683